src/js/editor.js

```diff
@@ -179,6 +179,8 @@ class ToastUIEditor {
       util.forEach(this.options.events, (fn, key) => this.on(key, fn));
     }
 
+    this.initialHtml = this.getInitialHtml();
```

Comment:
As we already have `setHTML`, `setValue`, I don't think we need another `set/getInitialHtml`.
Calling `setHTML` below is enough.

Reply:
Actually, I thought I might have to separate the setting/getting codes. Because if the codes that get html string using `innerHTML` be placed below line 184 (`this.layout = new Layout(...)`), it will return not only initial html text but also other html text (Perhaps it is a code of rendered editor ui.) 

I think if I don't use `set/getInitialHtml`, the code will be like this:
```js
...
this.initialHtml = this.options.el.innerHTML;
this.options.el.innerHTML = '';
this.layout = new Layout(options, this.eventManager);
...
this.setHtml(this.initialHtml);
...
```
Is there other way to solve this problem?

Reply:
Yea. Your initial thought isn't wrong at all. I was with you too.

The issue requested is to use `innerHTML` as the editor content when the editor starts up.
In fact, the issue brings another issue which we should set the content back to `innerHTML` when the editor ends.
And I think the consideration is a little out of the scope of the editor covers now.
Let's focus on just setting the content this time.

And the code seems good.
Just make the `initialHtml` local variable. It doesn't need to be a property.

Again. Nice job @ParkSB

Reply:
Thank you very much🙂 I'll make a new PR soon.